docs/pages/choose-an-edition/teleport-cloud/downloads.mdx

```diff
@@ -21,13 +21,19 @@ Binaries include:
 |`tsh`| A client tool for accessing resources within your cluster|
 |`tbot`| A client tool you will use to associate a bot user with a machine identity|
 
+<Notice title="macOS installations">
+For macOS the `tsh` is only signed for macOS in the Teleport client-only package.
```

Comment:
This first sentence is a little hard to read. Perhaps:

For macOS, `tsh` comes in the main Teleport package and also in a separate client-only package. The client-only package includes a signed variant of `tsh`. Certain functionality, such as Touch ID support, only works when `tsh` has been signed. Therefore, if you are installing the main Teleport package on a macOS system, we recommend you first install the main Teleport package and then install the client-only package over this.

Reply:
thanks @strideynet , updated